src/rpcclient.cpp

```diff
@@ -109,6 +109,10 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "z_gettotalbalance", 0},
     { "z_gettotalbalance", 1},
     { "z_gettotalbalance", 2},
+    { "z_mergetoaddress", 0},
```

Comment:
Remove `{ "z_mergetoaddress", 0},`.  Causes bug where argument like "*" causes JSON parsing errors (later in review).

Reply:
Yeah, it's annoying. I think the way to fix this properly is to remove this line, and then if the string doesn't match one of the expected constants, manually call the JSON parser.

Reply:
Follow-up question: Why does the RPC test I wrote not fail? It uses both string and array arguments...

Reply:
Ah, it's because this line affects how `zcash-cli` parses the CLI inputs. This is kind of annoying, because it means that if we remove this line, `zcash-cli` sends an unparsed JSON string representation as an actual string.

Okay, to fix this, I'm going to just make it always an array, but it can contain the keywords.